scripts/test/shared.py

```diff
@@ -493,18 +493,8 @@ def minify_check(wast, verify_final_result=True):
     print('      ', ' '.join(cmd))
     subprocess.check_call(cmd, stdout=open('a.wast', 'w'), stderr=subprocess.PIPE)
     assert os.path.exists('a.wast')
-    subprocess.check_call(WASM_OPT + ['a.wast', '--print-minified', '-all'],
+    subprocess.check_call(WASM_OPT + ['a.wast', '-all'],
                           stdout=open('b.wast', 'w'), stderr=subprocess.PIPE)
-    assert os.path.exists('b.wast')
-    if verify_final_result:
-        expected = open('a.wast').read()
-        actual = open('b.wast').read()
-        if actual != expected:
-            fail(actual, expected)
-    if os.path.exists('a.wast'):
-        os.unlink('a.wast')
-    if os.path.exists('b.wast'):
-        os.unlink('b.wast')
```

Comment:
It looks like asserting that b.wast exists would still be useful. Also, why delete the code that removes a.wast and b.wast after running the test?

Reply:
I don't think we need to remove files manually. Since @sbc100 moved us unto running tests in a test folder, things are contained anyhow.

What would checking b.wast existing be helpful? It'll definitely exist since we opened the file.

Reply:
Fair enough. Asserting that b.wast exists is mostly nice for symmetry since we do the same check on a.wast. I'd be ok with removing that check, too. We could also just not write a b.wast, since we don't use it for anything anymore.

Reply:
Good points, thanks. Done.